### PR TITLE
Hotfix/improve registration flow

### PR DESCRIPTION
**Pull request by CDavidSamuelsson**
#153  Make so the user get sent instantly to the start page after verifying the account instead of sending them to the login page

Note: The grade build version is set to 7.3.1 for now.

### CODE REVIEW

File: app/src/androidTest/java/com/flexicharge/bolt/TestVerifyActivity.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                "firstName" to "TESTAR",
+                "lastName" to "TESTARSSON"
+            )
+
+        activity.signIn(userInfo)
+
+
+        }
+
+         */
```
Comment (foal20ym) [inline app/src/androidTest/java/com/flexicharge/bolt/TestVerifyActivity.kt:50]: Remove commented code to preserve a clean codebase
```suggestion
```

File: app/src/androidTest/java/com/flexicharge/bolt/TestVerifyActivity.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        assert(2 == 2)
```
Comment (foal20ym) [inline app/src/androidTest/java/com/flexicharge/bolt/TestVerifyActivity.kt:51]: I don't think this necessary right? ;)
```suggestion
```

File: app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt [hunk 3]
```diff
@@ -697,6 +697,9 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback, ChargePointListAda
             setChargerButtonStatus(chargerInputStatus!!, false, getString(R.string.charger_status_enter_code), 3)
         }
     }
+    public fun addera() : Int {
+        return 4
+    }
```
Comment (foal20ym) [inline app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt:702]: I don't think this function is used or needed
```suggestion
```

File: app/src/test/java/com/flexicharge/bolt/ExampleUnitTest.kt [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
+
+
 class ExampleUnitTest {
     @Test
     fun addition_isCorrect() {
         assertEquals(4, 2 + 2)
     }
+
+
 }
```
Comment (foal20ym) [inline app/src/test/java/com/flexicharge/bolt/ExampleUnitTest.kt:25]: Test code should be removed before pushing.

**Review by foal20ym — CHANGES_REQUESTED** [review]
Looks very good overall. I just suggest a couple of small changes which is not related to your code or its functionality.

File: app/src/test/java/com/flexicharge/bolt/ExampleUnitTest.kt (hunk 4 above)
Comment (CDavidSamuelsson) [reply app/src/test/java/com/flexicharge/bolt/ExampleUnitTest.kt:25]: That is a test example from last year, but we should remove it anyway ?

Comment (foal20ym) [reply app/src/test/java/com/flexicharge/bolt/ExampleUnitTest.kt:25]: Sorry! I did not see that. You can leave it as it is. I will approve the pull request 👍

**Pull request merged**